src/com/boxboat/jenkins/library/git/GitRepo.groovy

```diff
@@ -10,6 +10,12 @@ class GitRepo implements Serializable {
 
     private shortHashLength = 12
 
+    static List<String> remoteBranches(String gitRemoteUrl) {
+        return Utils.resultOrTest(Config.pipeline.sh(returnStdout: true, script: """
+            git ls-remote --heads "${gitRemoteUrl}" | sed 's|.*refs/heads/\\(.*\\)|\\1|g'
+        """)?.trim()?.split('\n')?.findAll { it -> !it.isEmpty() }, ["master", "develop"])
```

Comment:
should the `master` or `develop` actually be a reference to a config.yaml option? What if they are using non-standard branch names?

Reply:
`["master", "develop"]` is the `Test` param for `Utils.resultOrTest`

Reply:
Ahh... my eyes skipped right over the `resultOrTest`